pallet-logion-loc/src/tests.rs

```diff
@@ -514,6 +544,27 @@ fn it_acknowledges_file_as_verified_issuer() {
     });
 }
 
+#[test]
+fn it_fails_to_close_with_file_unacknowledged_by_verified_issuer() {
+    new_test_ext().execute_with(|| {
+        setup_default_balances();
+        create_collection_and_nominated_issuer();
+        assert_ok!(LogionLoc::set_issuer_selection(RuntimeOrigin::signed(LOC_OWNER1), LOC_ID, ISSUER_ID1, true));
+        let file = FileParams {
+            hash: sha256(&vec![1, 2, 3]),
+            nature: sha256(&vec![4, 5, 6]),
+            submitter: SupportedAccountId::Polkadot(ISSUER_ID1),
+            size: 456,
+        };
+        assert_ok!(LogionLoc::add_file(RuntimeOrigin::signed(LOC_REQUESTER_ID), LOC_ID, file.clone()));
+
+        assert_err!(LogionLoc::close(RuntimeOrigin::signed(LOC_OWNER1), LOC_ID, None, true), Error::<Test>::CannotCloseUnacknowledgedByVerifiedIssuer);
+
+        let loc = LogionLoc::loc(LOC_ID).unwrap();
+        assert_eq!(loc.files[0], expected_file(&file, NOT_ACKNOWLEDGED, NOT_ACKNOWLEDGED));
```

Comment:
Could also check that `closed` flag is still false.

Reply:
Correct. Done 3x.